### PR TITLE
[OVERFLOW-482] Refactor Teams Page

### PR DESCRIPTION
**Pull request by kenorenzsun**
## Backlog Link
https://framgiaph.backlog.com/view/SUN_OVERFLOW-482
## Commands
None
## Pre-conditions
None
## Expected Output
- page should be the same design with figma
## Notes
None
## Screenshots
image
image
image

### CODE REVIEW

**Review by jules-lucero — DISMISSED** [review]

File: frontend/pages/teams/index.tsx [hunk 1]
```diff
@@ -96,61 +98,57 @@ const TeamsListPage = (): JSX.Element => {
...
-                        <form onSubmit={handleSearchSubmit}>
-                            <SearchInput
-                                placeholder="Search team"
-                                value={searchKey}
-                                onChange={onSearchInputChange}
-                            />
-                        </form>
-                        {isSearchResult && renderSearchResultHeader()}
-                    </div>
+        <div className="h-full w-full overflow-y-auto border border-neutral-200 bg-neutral-white p-4">
```
Comment (jules-lucero) [inline frontend/pages/teams/index.tsx]:
```suggestion
        <div className="h-full w-full overflow-y-auto border border-neutral-200 rounded-[5px] bg-neutral-white p-4">
```

Comment (kenorenzsun) [reply frontend/pages/teams/index.tsx]: I updated 🙇

**Review by jules-lucero — DISMISSED** [review]

**Comment by nicoleamber** [comment]
## QA Feedback

- Should have 9 cards per page
- Container div for content should hug the contents (see Figma design)
- Search results show total number during initial submit, but changes number after a second

Proof:
<details>
[Screencast 2023-04-28 17-00-27.webm](https://user-images.githubusercontent.com/116238730/235106984-5e5b05c9-c013-400e-9eae-b26105473955.webm)

</details>

**Comment by jules-lucero** [comment]
Please apply the correct scrollable container. Refer to `Questions` page or `Tags` page for the implementation

**Review by jules-lucero — APPROVED** [review]

**Review by nicoleamber — APPROVED** [review]

**Pull request merged**